app/components/Chart/utils.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+export interface DistributionDataPoint {
+  name: string;
+  count: number;
+}
+
+export const CHART_COLORS = [
```

Comment:
Would it be possible to include more chart colours? I have sometimes experienced charts being difficult to read because it ran out of colours.

Reply:
Yup, I added some more colors now, so that we now have 12 colors. That should suffice for all charts. ✅